content/docs/enterprise/branding/colors.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Colors
+description: Configure primary and secondary colors for the Enterprise Console and Error Pages.
+---
+Adding a [hex code](https://color.adobe.com/create/color-wheel) to the Primary Color setting will affect the majority of the theming on the console.
+For more details on how the types of colors will impact the look and feel of Pomerium, see [material ui](https://material.io/resources/color/#!/?view.left=0&view.right=0)'s color documentation.
+
+![Primary Color Pomerium Enterprise](img/branded_colors_console.png)
+
+There is also a secondary color that is used for a few things such as certain button colors.
+
+![Secondary Color Pomerium Enterprise](img/secondary_color.png)
+
+Choosing colors also affects the User Info and Error Pages that the Open Source serves.
```

Comment:
```suggestion
Color choice also affects the User Info and Error Pages. 
```
I'm not feeling strongly about this edit as it's very contextual. It also took me three reads to understand what I think is going on, and my edit reflects what I think was trying to be conveyed.

I took out "open source serves" because I wasn't sure what that contributed to the sentence. Aren't User Info and Error Pages in both OSS and Enterprise?

If it needs to go back in, I suggest: "Color choice also affects the User Info and Error Pages served by the Open Source version." for clarity